lib/ClimaLandSimulations/experiments/ozark.jl

```diff
@@ -1,6 +1,21 @@
 using ClimaLandSimulations
+using ClimaLandSimulations.Fluxnet
 
-sv, sol, Y, p, inputs, climaland = fluxnet_simulation("US-MOz");
+# default parameters, except for custom parameter hetero_resp.b
+sv_test, sol_test, Y_test, p_test = fluxnet_simulation(
```

Comment:
maybe add a comment that this is where the simulation is actually solved (or rename fluxnet_simulation -> run_ fluxnet_simulation?)

Reply:
would run_fluxnet be a better name? (run_fluxnet_simulation becomes a bit long)

Reply:
that sounds good to me!